pkg/scheduler/factory/factory.go

```diff
@@ -1076,7 +1081,7 @@ func (c *configFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	// Init equivalence class cache
 	if c.enableEquivalenceClassCache {
 		c.equivalencePodCache = equivalence.NewCache()
-		glog.Info("Created equivalence class cache")
+		glog.Infof("Created equivalence class cache")
```

Comment:
Since we are not formatting anything here, glog.Info is the right function to use.